package.json

```diff
@@ -89,12 +86,14 @@
     "karma-phantomjs-launcher": "^1.0.2",
     "karma-sourcemap-loader": "^0.3.7",
     "karma-webpack": "^1.8.0",
+    "lodash.omit": "^4.5.0",
```

Comment:
Underscore has omit which can be used instead of pulling this in.

Reply:
We don't have underscore in this library.

Reply:
Touche